vulnerabilities/importers/fireeye.py

```diff
@@ -44,7 +44,7 @@ def advisory_data(self) -> Iterable[AdvisoryData]:
                 if Path(file).stem == "README":
                     continue
                 try:
-                    with open(file) as f:
+                    with open(file, encoding="utf-8-sig") as f:
```

Comment:
@harsh098 thanks for your PR, please add tests

Reply:
ok @TG1999

Reply:
Should I add this test for `md_list_to_dict` or `parse_advisory`.
I am currently going for `md_list_to_dict` as the goal to accomplish is to check if
`md_list_to_dict` works with the `utf-8-sig` encoding or not